survey-runner.tf

```diff
@@ -37,7 +37,8 @@ module "survey-runner-on-beanstalk" {
   dns_zone_name = "${var.dns_zone_name}"
   deployment_policy = "${var.eb_deployment_policy}"
   rolling_update_enabled = "${var.eb_rolling_update_enabled}"
-  secrets_file_name       = "${var.survey_runner_secrets_file_name}"
+  secrets_file_name      = "${var.survey_runner_secrets_file_name}"
+  respondent_account_url = "${var.respondent_account_url}"
```

Comment:
where does `"${var.respondent_account_url}"` come from?

Reply:
From the global.vars.tf? I thought

Reply:
The `global_vars.tf` that you have modified is in a separate module (folder). You will need to add this to the developer defaults. if you try running this branch its clear that it is missing